### PR TITLE
3.8.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-3.8.0-release](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-3.8.0-release)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.8.0 (2020-02-18)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Should we update the date? Or is yesterday's date fine?

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-3.8.0-release](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-3.8.0-release)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done.

**Pull request merged**